pyalgotrading/algobulls/connection.py

```diff
@@ -47,7 +46,7 @@ def set_access_token(self, access_token):
         assert (isinstance(access_token, str) is True), f'Argument "access_token" should be a string'
```

Comment:
`assert (isinstance(access_token, str) is True)`

`isintance()` already returns True or False

so `assert isinstance(access_token, str)` should suffice to ascertain the truth value

kindly check and if confirmed, make changes everywhere

Reply:
to summarize:

try

`        assert isinstance(access_token, str), f'Argument "access_token" should be a string'`

instead of 

`        assert (isinstance(access_token, str) is True), f'Argument "access_token" should be a string'`

and if it works, make the change everywhere